libobs/obs-audio.c

```diff
@@ -516,6 +517,9 @@ bool audio_callback(void *param, uint64_t start_ts_in, uint64_t end_ts_in,
 		obs_source_audio_render(source, mixers, channels, sample_rate,
 					audio_size);
 
+		if (source->audio_pending && source->audio_ts)
```

Comment:
While chasing a different audio bug, I found that the changes in this file appear to negatively affect A-V sync for some jittery sources. Due to the nature of how difficult it is to reproduce this bug, I am unable to practically investigate any further.